activities/communication/pitching.md

```diff
@@ -0,0 +1,145 @@
+---
+type: resource
+---
+
+# Pitching the Foundation for Public Code
+
+Staff often find themselves needing to pitch the Foundation for Public Code to diverse audiences, which can be daunting if you've never done it before. We've collected pitches we already use so it's easier to practice.
+
+## Audiences
+
+These are the audiences we've spoken to about the Foundation for Public Code:
+
+* managers at a public organization
+* general policy experts or civil servants
+* developers at a public organization
+* digital or tech people who work for a public organization
+* general developers and other digital or tech people
+* open source enthusiasts
+* 'open' enthusiasts
+* vendors to government
+* the general public
+* politicians and politicians' assistants or special political advisors (SPADs)
+* journalists
+* activists
+* philanthropists
+
+## Opening lines
+
+These are pitches we already use.
+
+For developers, tech people and open source enthusiasts:
+
+* "You know the Free Software Foundation Europe's (FSFE) [Public Money, Public Code] campaign? We're helping public organizations actually do it."
+
+For people with government experience or interest:
+
+* "We're helping public organizations that develop their own software to collaborate and share it with other public organizations so that the public doesn't need to duplicate the work."
+* [Boris pitches]
```

Comment:
* **Digital transformation**. "Every government is going through digital transformation but they are way more governments in the world that are going through digital transformations than there are technologists that understand that or programmers that can help with that. We need to be able to work together, it's either we work together or just we give keys to companies that'll sell us things, and then they'll keep us in their grid. So we need to learn how to work together and as the Foundation we are trying to build that collaboration at scale."
* **Mutual self help institution**. "We help public organizations to work together in order to go through digital transformation in a way they control and they are able to deal with. However, it's in none of their mandates to help others, yet real collaboration doesn't ever happen if you don't help each other and I think the Foundation for Public Code adds a player to that field where everyone can help each other by helping themselves."
* **Open source for public organizations**. "We are an open source foundation like you have the Eclipse Foundation or the Linux Foundation but then for public organizations. Public organizations are quite different so they need a different approach, but they can still benefit from that type of collaboration or trying to figure out how that works."
* **Software collaboration is better**. "We envision a future where software, as part of government, is taken a seriously as policy is now, dus being open becomes really crucial. That comes with the added advantage that you can then be critical with each others' open software and you can work together to make everyone's software better."

Reply:
Done! Thank you!